source/classification1.md

```diff
@@ -599,11 +599,8 @@ distance using the formula above: we square the differences between the two obse
 and concavity coordinates, add the squared differences, and then take the square root.
 In order to find the $K=5$ nearest neighbors, we will use the `nsmallest` function from `pandas`.
 
-> **Note:** Recall that in the {ref}`intro` chapter, we used `sort_values` followed by `head` to obtain
-> the ten rows with the *largest* values of a variable. We could have instead used the `nlargest` function
-> from `pandas` for this purpose. The `nsmallest` and `nlargest` functions achieve the same goal 
-> as `sort_values` followed by `head`, but are slightly more efficient because they are specialized for this purpose.
-> In general, it is good to use more specialized functions when they are available!
+```{index} nsmallest
```

Comment:
What does this do? I don't see any reference or anything show up in the textbook 
![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/1bc508a3-7820-4035-9447-78c2e64a5934)

Reply:
@joelostblom it's for creating index entries in the PDF version

Reply:
(it doesn't do anything in HTML currently -- but perhaps there's a way to produce an index page?)